app/validators/embargo_date_validator.rb

```diff
@@ -6,6 +6,27 @@ class EmbargoDateValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     return if value.nil?
 
+    collection = Collection.where(id: record.model[:work][:collection_id]).first
+    return if collection.blank? || collection.release_duration.blank?
+
+    valid_embargo_value(record, attribute, value) if valid_embargo_collection(collection, record, attribute, value)
```

Comment:
Only calls the method if the collection release duration fails preventing duplicate errors if the collection's release duration is `3 years`